test/suites/Tests-15-ECMA.js

```diff
@@ -578,7 +578,7 @@ define(['pumascript', 'esprima'], function (puma, esprima) {
         var result = puma.evalPuma("Array.prototype.constructor;");
         result.makeValue();
         equal(result.success, true, "Passed!");
-        equal(result.value.toString(), c, "Passed!");
+        equal(result.value.toString().replace(/\r?\n/g, ' '), c, "Passed!");
```

Comment:
It would be nice to try to be consistent about the usages of single quotes and double quotes.